homeassistant/components/ecobee/const.py

```diff
@@ -20,8 +20,9 @@
 
 DOMAIN = "ecobee"
 DATA_ECOBEE_CONFIG = "ecobee_config"
+DATA_HASS_CONFIG = "hass_config"
```

Comment:
We should store the data under an integration specificially named key. Normally we store everything under the `DOMAIN` key and use a dataclass or a dict if more than one thing needs to be stored. It's ok for now though, if we want to avoid a larger refactor, to just rename the key to something integration specific.

Reply:
```suggestion
DATA_HASS_CONFIG = "ecobee_hass_config"
```